ProcessLib/TES/TESProcess.cpp

```diff
@@ -175,50 +163,79 @@ void TESProcess::initializeConcreteProcess(
         mesh.getDimension(), mesh.getElements(), dof_table, integration_order,
         _local_assemblers, _assembly_params);
 
-    // secondary variables
+    initializeSecondaryVariables();
+}
+
+void TESProcess::initializeSecondaryVariables()
+{
+    // adds a secondary variables to the collection of all secondary variables.
     auto add2nd = [&](std::string const& var_name, unsigned const n_components,
                       SecondaryVariableFunctions&& fcts) {
-        this->_secondary_variables.addSecondaryVariable(var_name, n_components,
-                                                        std::move(fcts));
+        _secondary_variables.addSecondaryVariable(var_name, n_components,
+                                                  std::move(fcts));
     };
+
+    // creates an extrapolator
     auto makeEx =
         [&](std::vector<double> const& (TESLocalAssemblerInterface::*method)(
             std::vector<double>&)const) -> SecondaryVariableFunctions {
         return ProcessLib::makeExtrapolator(getExtrapolator(),
                                             _local_assemblers, method);
     };
 
-    add2nd("solid_density", 1,
-           makeEx(&TESLocalAssemblerInterface::getIntPtSolidDensity));
+    // named functions: vapour partial pressure ////////////////////////////////
+    auto p_V_fct = [=](const double p, const double x_mV) {
+        const double x_nV = Adsorption::AdsorptionReaction::getMolarFraction(
+            x_mV, _assembly_params.M_react, _assembly_params.M_inert);
+        return p*x_nV;
+    };
+    _named_function_caller.addNamedFunction(
+        {"vapour_partial_pressure",
+         {"pressure", "vapour_mass_fraction"},
```

Comment:
I think I aligned it with clang-format. The brace on the second line is nested into the one on the first line.